docs/ReleaseHistory.md

```diff
@@ -17,6 +17,7 @@
 - RUL: Add `DAT101/003.IPv6` non-sensitive data classification.
 - RUL: Add `DAT101/004.Integer` non-sensitive data classification.
 - RUL: Add `DAT101/005.Float` non-sensitive data classification.
+- RUL: Add `SEC101/055.Pkcs12CertificatePrivateKeyBundle` detection. This is currently solely regex based and does not yet guarantee fully-formed PKCS #12.
```

Comment:
>055 [](http://example.com/codeflow?start=19&length=3)

hey! can we fix the brain dead iterations == 1000 issue you came across? that's just wrong, the code was never intended to run those tests 1000 times on a permanent basic.

yes, we have a footgun here, and I'm glad to go remove it, since i'm the guy who added it